src/plugin.js

```diff
@@ -650,7 +643,7 @@ var zoomPlugin = {
           delete scaleOptions.max;
         }
       });
-      chartInstance.update();
+      chartInstance.update('none');
```

Comment:
Why `none` mode here? I think it would probably be ok to use `zoom` here as well.

Reply:
That might be a good idea. Lets leave it out from this pr still though and just use the `none` for now, because that probably meets most use cases.